src/descriptor/satisfied_constraints.rs

```diff
@@ -870,8 +870,8 @@ impl<'stack> Stack<'stack> {
         }
     }
 
-    /// Helper function to evaluate a After Node. Takes no argument from stack
-    /// `n CHECKSEQUENCEVERIFY 0NOTEQUAL` and `n CHECKSEQUENCEVERIFY`
+    /// Helper function to evaluate a Older Node. Takes no argument from stack
```

Comment:
I don't think you mean to change `After` to `Older` here; ditto in the other function's comment.